docs/faq/general.txt

```diff
@@ -71,6 +71,24 @@ Django was originally developed at World Online, the Web department of a
 newspaper in Lawrence, Kansas, USA. Django's now run by an international
 `team of volunteers <https://www.djangoproject.com/foundation/teams/>`_.
 
+How is Django licensed?
+=======================
+
+Django is distributed under `the BSD license
+<https://github.com/django/django/blob/master/LICENSE>`_. This is an
+open-source license granting you broad permissions to modify and
+redistribute Django.
+
+Why does Django include two license files?
```

Comment:
You might say "Python's license file" instead of "two license files" (or "two license files in the root of the repository"). A pedantic reader might note that there are license files in other places, such as for jQuery.